lib/config/fileconf.go

```diff
@@ -1538,6 +1540,12 @@ type AccessGraphAWSSync struct {
 	ExternalID string `yaml:"external_id,omitempty"`
 }
 
+// AccessGraphAzureSync represents the configuration for the Azure AccessGraph Sync service.
+type AccessGraphAzureSync struct {
+	SubscriptionID string `yaml:"subscription_id,omitempty"`
+	Integration    string `yaml:"integration,omitempty"`
```

Comment:
Missing godoc.

Reply:
Going to actually remove `Integration` from fileconf since it's not used there. But will add godoc for `SubscriptionID`.

Reply:
The `Integration` can be used here.
Using Integration credentials and using DiscoveryConfig are unrelated features.
You can use a DiscoveryConfig using ambient credentials and you can also use an Integration for a DiscoveryService's Matcher.

Reply:
Interesting, wonder why AWS doesn't have it here either? I noticed it doesn't get set for file-configured matchers. I assumed it meant that only dynamic matchers use this field.

Reply:
I don't think it's a major issue, but we do support it. 
Given that AWS Sync doesn't have it, I think it's fine.

In theory, we just need to expose the field.
As you can see from the AWSMatcher (not related to TAG)
https://github.com/gravitational/teleport/blob/ef7a93fa3d4dda1ae0a65ae6e08aa5c3c1612547/lib/config/fileconf.go#L1701